CMakeLists.txt

```diff
@@ -34,10 +34,25 @@ file(GLOB UTILS_SRC "utils/*.c")
 file(GLOB PQ_HEADERS "pq-crypto/*.h")
 file(GLOB PQ_SRC "pq-crypto/*.c")
 
-if(S2N_NO_PQ)
+if (DEFINED ENV{S2N_ADDRESS_SANITIZER})
+    set(CMAKE_BUILD_TYPE "ASAN")
+endif()
+
+if (CMAKE_BUILD_TYPE STREQUAL "ASAN")
+    message(STATUS "Address sanitizer mode")
+endif()
+
+if (NOT CMAKE_BUILD_TYPE STREQUAL "Release")
+    message(STATUS "Copying test pems to build.")
+    file(GLOB TEST_PEM_FILES "tests/pems/*.pem")
+    file(COPY ${TEST_PEM_FILES} DESTINATION "pems")
+endif ()
+
+if(S2N_NO_PQ OR CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL "aarch64")
```

Comment:
Was there an error that you ran into with PQ code on aarch64?

Reply:
Initially, yes- going back to capture the specific test can no longer reproduce it...will remove.